src/cli.ts

```diff
@@ -35,7 +41,25 @@ if (Deno.args.length < 1 || args.help || args.h) {
 
 const url = args._[0];
 let path = args.out;
-const headers = parseHeaders(args.H);
+
+// Combine H and Hapikey arguments into a single list of headers
+const combinedHeaders = [...(args.H || [])].filter((header): header is string =>
+  typeof header === "string"
+);
+
+if (args.Hapikey) {
+  combinedHeaders.push(args.Hapikey);
+}
+
+const headers = parseHeaders(combinedHeaders);
```

Comment:
I don't think you want to mix the global header and regular ones, as I can see you later filter it out. Please juste parse and pass those separately

Reply:
adding the combined header because they should be used in api call, right?